src/browsers.js

```diff
@@ -217,6 +217,9 @@ const SauceBrowsers = {
   },
 
   _normalize: (data) => {
+    if (!Array.isArray(data)) {
+      throw new Error(`Invalid Response: ${data}`);
```

Comment:
we need to look at loggers in our components because if that data comes back with something as
```{
  code: 1456,
  message: Failed to initialize Sauce tunnels because of <insert reason>
  values: {]
}```
,
throwing an error like `throw new Error('Invalid Response: ${data}');` will print `[Error: Invalid Response: [object Object]]`

Reply:
Great call!